napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -733,156 +772,180 @@ def filter(self, text: str, starts_with_chars: int = 1):
                 item.setHidden(False)
 
 
-class RefreshState(Enum):
-    REFRESHING = auto()
-    OUTDATED = auto()
-    DONE = auto()
-
-
 class QtPluginDialog(QDialog):
-    def __init__(self, parent=None) -> None:
+    def __init__(self, parent=None, prefix=None) -> None:
         super().__init__(parent)
 
         self._parent = parent
         if (
-            getattr(parent, '_plugin_dialog', None) is None
-            and self._parent is not None
+            parent is not None
+            and getattr(parent, '_plugin_dialog', None) is None
         ):
             self._parent._plugin_dialog = self
 
-        self.refresh_state = RefreshState.DONE
         self.already_installed = set()
         self.available_set = set()
+        self._prefix = prefix
 
         self._plugin_data = []  # Store plugin data while populating lists
-        self.all_plugin_data = []  # Store all plugin data
+        self._all_plugin_data = []  # Store all plugin data
```

Comment:
Is this more like a queue of pending plugins? Let's rename it accordingly 🙏

Reply:
Fixed